team_red: ["jnogueira","tmadeira","talmeida","jmoreira"]
team_green: ["ttavares","acastro","dsilva","psilva"]
team_blue: ["drato","moliveira","jsantos","mferreira"]